Jibril/BinaryTree/week_May_16_22/city_paths.py

```diff
@@ -0,0 +1,72 @@
+"""
+                    Seattle
+                  /        \
+              Denver     New York
+            /      \           \
+        Miami   Los Angeles   Boston
+                 /
+             Dallas
+                \
+            Minneapolis
+
+1. Find the path from Seattle to Dallas?
+2. Find the distance from Seattle to Minneapolis?
+3. Find the distance from Seattle to Dallas?
+
+"""
+
+
+class TreeNode:
+  def __init__(self, val, distance):
+    self.val = val
+    self.distance = distance
+    self.right = None
+    self.left = None
+
+def path_cities(root, city):
```

Comment:
You didn't add iterative solution here as well.

Reply:
Added the iterative solution on this one but it still has some bugs. I will see if you can find the iterative solution for this one by tomorrow before our session but I would like to see how you went about solving it iteratively.